.gitignore

```diff
@@ -10,3 +10,4 @@
 **/data/*
 
 **/env
+*sandbox*.py
```

Comment:
I made a script to test out the dataset creation against MACH -- namely to validate matrix shapes, I figured we can keep this as a testbed that doesn't get committed, I can also take this out if it's creating noise.